docs/source/examples.md

```diff
@@ -44,7 +44,7 @@ python sample.py -n 103 --azureml
 then the function `submit_to_azure_if_needed` will perform all the required actions to run this script in AzureML and exit. Note that:
 
 * code after `submit_to_azure_if_needed` is not run locally, but it is run in AzureML.
-* the print statement prints to the AzureML console output and is available in the `Output + logs` tab of the experiment in the `70_driver_log.txt` file, and can be downloaded from there.
+* the print statement prints to the AzureML console output and is available in the `Output + logs` tab of the experiment in the `70_driver_log.txt` if using the old AzureML runtime, or `std_log.txt` if using the new AzureML runtime, and can be downloaded from there.
```

Comment:
I'm curious. How does one know which runtime is using?

Reply:
So far the only way I've been able to tell is a notification at the top of the page while the run is running.

Reply:
Thanks! @ant0nsc, do you know more about this?